orchestrator/jobsplanner.go

```diff
@@ -3,11 +3,11 @@ package orchestrator
 import (
 	"context"
 	"fmt"
+	"github.com/streamingfast/substreams/pipeline/store"
```

Comment:
What’s the difference between `pipeline/store/` and `state/` ?

I understand that `state/` should have long been renamed to `store` .. which would make this new package irrelevant.

Let's say we rename `state` to `store`, then we could have the Accessor concept in `state`. More on that later.